curator/Dockerfile

```diff
@@ -22,7 +22,8 @@ RUN if [ -n "${LOCAL_REPO}" ] ; then \
      curl -s -o /etc/yum.repos.d/local.repo ${LOCAL_REPO} ; \
     fi
 
-RUN INSTALL_PKGS="elastic-curator-${CURATOR_VER} \
+RUN INSTALL_PKGS="python2-pyyaml-3.12-11 \
```

Comment:
Do we need to specify the version if it is the latest one we have?

Reply:
It depends.  For downstream, if this package is coming from a RHEL repository, and there are multiple versions of the package, then yes, we need to specify the version.  If this package is coming from a RHAOS compose, then no, because there can be only 1 package version per compose.